jetty-ee10/jetty-ee10-servlet/src/test/java/org/eclipse/jetty/ee10/servlet/ServletContextHandlerTest.java

```diff
@@ -2432,4 +2432,35 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
         String setCookieValue = response.get(HttpHeader.SET_COOKIE);
         assertThat(setCookieValue, containsString("example=bogus; SameSite=Strict"));
     }
+
+    @Test
+    public void testEmptyPathInfo() throws Exception
+    {
+        ServletContextHandler context = new ServletContextHandler(null, "/c1", ServletContextHandler.NO_SESSIONS);
+        context.setAllowNullPathInContext(true);
+        context.addServlet(new ServletHolder("default-servlet", new HttpServlet()
+        {
+            @Override
+            protected void service(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException
+            {
+                resp.setContentType("text/plain");
+                resp.setCharacterEncoding("UTF-8");
+                resp.getWriter().write("OK2\n");
+                resp.getWriter().close();
+            }
+        }), "/");
+
+        _server.setHandler(context);
+        _server.start();
+        String rawRequest = """
+            GET /c1 HTTP/1.1\r
+            Host: localhost\r
+            Connection: close\r
+            \r
+            """;
+
+        String rawResponse = _connector.getResponse(rawRequest);
+        HttpTester.Response response = HttpTester.parseResponse(rawResponse);
+        assertThat(response.getContent(), containsString("OK"));
```

Comment:
```suggestion
        assertThat(response.getContent(), containsString("OK2"));
```

shouldn't we test "OK2" as this is what the servlet writes?